SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -54,6 +54,7 @@ Patch3:         CVE-2024-3727.patch
 #Note (mfrw): The patch for CVE-2024-37298 only applies to podman.
 Patch4:         CVE-2024-37298.patch
 Patch5:         CVE-2024-6104.patch
+Patch6:         CVE-2024-24786.patch
```

Comment:
It seems this file is composed of a few different projects that need to patched.
We have to explicitly call  `patch`